### PR TITLE
Adds sub-routes in admin/content & adds ui for content/social-links

### LINKED ISSUE #195: Create all subroutes under admin/content and ui for admin/content/social-links only
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

screenshot from 2017-06-09 18-23-49

- Create all the sub routes under admin/content
- Create the UI for admin/settings/system/social-links

### PR DESCRIPTION
**Pull request by geekyd**
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Adds all sub-routes in admin/content
- Adds ui for content/social-links
- Adds navigation in admin/content

Deployment link: https://open-event-frontend-app.herokuapp.com/admin/content
Fixes #195 
@niranjan94 @CosmicCoder96 please review.

### CODE REVIEW

**Comment by geekyd** [comment]
@niranjan94 all the fields will be required?

**Comment by niranjan94** [comment]
None of them are required. Just make sure they accept only valid URLs

**Comment by geekyd** [comment]
@niranjan please review. Thanks!

File: app/templates/admin/content/index.hbs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<form class="ui form">
+  <div class="field">
+    <label>{{t 'Support'}}</label>
+    {{input type='url' name='support'}}
```
Comment (geekyd) [inline app/templates/admin/content/index.hbs]: `type='url'` will ensure that the entered input is valid or not.

Comment (geekyd) [inline app/templates/admin/content/index.hbs]: `type='url'` will ensure that the entered input is valid or not.

**Comment by geekyd** [comment]
@niranjan Please review. Thanks

**Comment by geekyd** [comment]
@niranjan94 Please review.
Deployment link: https://open-event-frontend-app.herokuapp.com/admin/content

File: app/templates/components/forms/admin/content/social-links-form.hbs [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<form class="ui form" {{action 'submit' on='submit'}}>
```
Comment (niranjan94) [inline app/templates/components/forms/admin/content/social-links-form.hbs]: Let this class and tag be part of component definition itself.

**Comment by geekyd** [comment]
@niranjan94 Made the changes, please review.

File: app/components/forms/admin/content/social-links-form.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+              prompt : this.i18n.t('Please enter a valid url')
+            }
+          ]
+        }
+      }
+    };
+  },
+
+  actions: {
+    submit() {
```
Comment (niranjan94) [inline app/components/forms/admin/content/social-links-form.js]: Move this outside `actions` so that the event gets binded automatically

Comment (geekyd) [inline app/components/forms/admin/content/social-links-form.js]: Done. :)

**Comment by geekyd** [comment]
@niranjan94 @CosmicCoder96 Please review. Thanks!

**Comment by geekyd** [comment]
@niranjan94 @CosmicCoder96 The form validation has stopped working after adding `tagName` to the component & adding `ui form` class, but when we have the `<form></form>` inside the handlebars, it works fine.

The reason behind this
Inside the mixin we have a variable called `$form` which is used to hold the form object, used in `onValid()` of the mixin. In the current approach the line `this.getForm()` returns an empty array, & not the form element. As a result the validation never takes place & hence fails.

I would suggest that we should use a common style. If we desire to use the current approach we must make some changes to the mixin.

**Comment by CosmicCoder96** [comment]
@geekyd I agree. I too noticed that after I made the same changes on my PR after seeing @niranjan94's comments on your PR and so have reverted back for now.

**Comment by geekyd** [comment]
@CosmicCoder96 @niranjan94 Updated the PR. Reverted to the old approach. Please review. 
Deployment Link: https://open-event-frontend-app.herokuapp.com/admin/content

**Pull request merged**